src/tools/pontoon/translation_workspace.md

```diff
@@ -192,6 +193,21 @@ Icons to the right indicate the [status](#translation-status) of each translatio
 
 In the screenshot above, the first item is the approved translation (green checkmark), while the other two are rejected suggestions.
 
+#### Status banners
+
+Status banners provide immediate context about a user to other community members. Banners indicate either the role of the user within the current project and locale, if they are administrators, or if they are new to Pontoon. Banners are displayed at the bottom of a user's profile picture.
+
+![Collage of all 5 possible status banners](../../assets/images/pontoon/translation_workspace/status_banners.png "Screenshot displaying a collage of all 5 possible status banners that can be displayed")
+
+The available status banners are:
+* `MNGR` for team managers.
+* `TRNSL` for translators.
+* `PM` for [project managers](teams_projects.md#project-page).
+* `ADMIN` for users with site-wide admin permissions.
```

Comment:
```suggestion
* `ADMIN` for [administrators](users.md#user-roles).
```

I'd also link `project managers`, `translators` and `team managers` to the [User roles](users.md#user-roles) page.